### PR TITLE
[CoreBundle] Enhance ws user tool csv

### PR DESCRIPTION
**Pull request by ptsavdar**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Fixed tickets | 

- Search also by username in WS user list
- Display success message on CSV user import in WS
- Disable contextmenu -> download for Image, Video, Pdf
- Improve home page performance

### CODE REVIEW

**Comment by LaurentGruber** [comment]
@ptsavdar tout fonctionnera également pour les espaces ouvert aux anonymes ?

File: plugin/image-player/Resources/modules/resources/image/selectors.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import {createSelector} from 'reselect'
 
 const image = state => state.image
+const resourceNode = state => state.resourceNode
```
Comment (Elorfin) [inline plugin/image-player/Resources/modules/resources/image/selectors.js]: This selector already exists in `#/main/core/layout/resource/selectors`.

**Comment by ptsavdar** [comment]
@LaurentGruber Normalement oui. J'ai pas change qqch impactact sur la vue en anonymes (de tt facon les anonymes n'ont pas acces a la liste des utilisateurs d'un WS)

File: plugin/image-player/Resources/modules/resources/image/selectors.js [hunk 2]
```diff
@@ -12,8 +13,14 @@ const hashName = createSelector(
   (image) => image.hashName
 )
 
+const canDownload = createSelector(
+  [resourceSelect.resourceNode],
+  (resourceNode) => resourceNode.rights.current.export
```
Comment (Elorfin) [inline plugin/image-player/Resources/modules/resources/image/selectors.js]: Sorry, last change : you should use `resourceSelect.currentRights`. Always use the most precise selector available, so if the underlaying structure changes, you will have nothing to change.

Comment (Elorfin) [reply plugin/image-player/Resources/modules/resources/image/selectors.js]: In fact, this whole selector should be in `#/main/core/layout/resource/selectors`

Comment (ptsavdar) [reply plugin/image-player/Resources/modules/resources/image/selectors.js]: Yes, my bad! I've just realised how it should be implemented properly. Thanks for your advice!

**Pull request merged**